src/tokens/ERC4626.huff

```diff
@@ -77,7 +76,7 @@
     [INITIAL_DOMAIN_SEPARATOR] sstore       // [decimals]
 
     // Copy the runtime bytecode with constructor argument concatenated.
-    0xa2                                    // [offset, decimals] - constructor code size
+    __codesize(CONSTRUCTOR)                 // [offset, decimals]
```

Comment:
:')

Reply:
something to be aware of is that this will only work if the inheriting constructor is called constructor

Reply:
Indeed good point, it needs to be taken into account.
What about documenting the way to use huffmate in the readme and especially contract constructors ?
It seems there was already talks in discord on how to import contracts. 
Having this + a warning on the name of the inheriting constructor would be ok ?

Adding a comment also in the constructor of ERC4626 could make sense but not really scalable and might be repetitive if other constructors make use of `__codesize`

Reply:
@cheethas bumping this  
What are your thoughts on this ?
If you think this is not necessary we can merge like this

Reply:
hey my apologies, ive been off the ball. Lets get it merged